### ISSUE TITLE
accessModeSufficient link pointed to wrong link

### ISSUE THREAD

**Issue by Jeffxz** [mattgarrish/epub-revision#2298]
The accessModeSufficient link in following line is pointing to [accessibilityHazard](https://schema.org/accessibilityHazard)

https://github.com/w3c/epub-specs/blob/main/epub33/a11y/index.html#L340

The correct link I think should be https://schema.org/accessModeSufficient

**Comment by mattgarrish** [comment]
Thanks, yes, looks like a copy/paste error!

I'll open a PR to fix shortly.

**Comment by Jeffxz** [comment]
Thanks!
